var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -58,9 +67,11 @@ def cmake_args(self):
             '-DDoublePrec_CPU=%s' % ('+double' in self.spec),
             '-DDoublePrec_GPU=%s' % ('+double-gpu' in self.spec),
         ]
+        # added cuda arch for p100s
         if '+cuda' in self.spec:
             args += [
                 '-DCUDA=on',
+                '-DCUDA_ARCH=60'
```

Comment:
so one can't to use any other arch? Looks like a wrong approach. You should probably instead add
```
variant('cuda_arch', default=None,
             description='CUDA architecture',
             values=('20', '30', '32', '35', '50', '52', '53', '60', '61',
                     '62', '70'),
             multi=True)
```
and use it
```
cuda_arch = spec.variants['cuda_arch'].value
if cuda_arch is not None:
```

Reply:
Thanks!

Reply:
p.s. this PR aims to unify a bit CUDA in Spack https://github.com/spack/spack/pull/6070/